docs/fuzz-targets.md

```diff
@@ -172,3 +172,39 @@ flag, so that only the most important parameters are discussed here.
 | `--sync`                                                                | Enables synchronous fuzzing. **May only be used for entirely synchronous code**.                                                                                                                                                                                                                         |
 | `-h`, `--custom_hooks`                                                  | Filenames with custom hooks. Several hooks per file are possible. See further details in [docs/fuzz-settings.md](docs/fuzz-settings.md).                                                                                                                                                                 |
 | `--help`                                                                | Detailed help message containing all flags.                                                                                                                                                                                                                                                              |
+
+## Coverage report generation
+
+To generate a coverage report, add the `--coverage` flag to the npm command. In
```

Comment:
Do you mean adding the `--coverage` flag to Jazzer.js CLI?

Reply:
Yes, changed.